conda/environments/cuml_dev_cuda10.1.yml

```diff
@@ -28,6 +28,8 @@ dependencies:
     - sphinx_markdown_tables
     - git+https://github.com/dask/dask.git@main
     - git+https://github.com/dask/distributed.git@main
+    - git+https://github.com/dask/dask-glm@main
+    - "dask[array] @ git+https://github.com/dask/dask.git@main"
```

Comment:
Why is this needed?

Reply:
This was necessary due to some dependency issues with the new pip resolver picking up an incompatible dask[array] by default even if dask is pinned to main.